test/views/returns/credits/CreditsClaimedListViewSpec.scala

```diff
@@ -46,7 +49,7 @@ class CreditsClaimedListViewSpec extends ViewSpecBase with ViewAssertions{
   "View" should {
 
     "have a header" in {
-      createView(form).select("h1").text mustBe "Do you want to claim tax back as credit for 1 April 2022 to 31 March 2023?"
+      createView(form).select("h1").text mustBe messages("creditsSummary.title-heading")
```

Comment:
here, should we prefer test the actual text over key? or Maybe both? I seen this as kind of integration test that verify that our page contain the right content (so the test became our SoT )as we do not do that at browser level.

Reply:
Content is fluid right now :man_shrugging: